share/commands/agent-factory-reset

```diff
@@ -62,6 +62,17 @@ while getopts "iIvdqcTf" opt; do
   esac
 done
 
+while true; do
+    echo "This command will permanently delete your node uuid and keys"
```

Comment:
We need a real message, this is kind of a placeholder, can you suggest a message ?